### DISCUSSION TITLE
`core` in raw json settings

### DISCUSSION THREAD

**Discussion by Shluffy** [ScratchAddons/ScratchAddons#3101] [Help] [answered]
`"core":{"lightTheme":false,"version":"1.7.1"}`
If you change it to `"core":{"lightTheme":false,"version":"1.0.0"}` it makes no noticeable difference. What is this doing here?

**Comment by apple502j** [comment, marked as answer]
`version` is not used by the extension. It is for humans reading the file.

**Reply by WorldLanguages** [reply]
And "just in case" we need it in the future to change how the import works depending on the version it comes from.

**Reply by Shluffy** [reply]
alright thanks. It would be cool to read that and set the extension version to that, and include addons from that (using version added) as well as any bugs in there
